### DISCUSSION TITLE
Tasmota + relay + DTH11

### DISCUSSION THREAD

**Discussion by TheWickerman69** [arendst/Tasmota#21249] [Q&A]
Hello everyone, I have a WT32-Eth01 with a relay running on timers to make a small fan cool my computer rack. So far so good, now I have incorporated a DTH11 and I can see the temperature.
I have thought about creating a rule that if the temperature inside rises above 30ºC the fan will be activated until it drops to 25ºC but I am going crazy.
I have copied what is said here https://tasmota.github.io/docs/Rules/#simple-thermostat-example
but I only get the relay to turn off every minute regardless of the temperature.
Someone gives me a hand?

**Comment by valdus2** [comment]
Something simple as: `On tele-DTH11#Temperature>30 do Power1 1 ENDON On tele-DTH11#Temperature<25 do Power1 0 ENDON`
(replace with your naming)
